compose/project.py

```diff
@@ -13,6 +14,8 @@
 
 log = logging.getLogger(__name__)
 
+VolumeFrom = namedtuple("VolumeFrom", 'source mount_option')
```

Comment:
(We should really move all those named tuples into their own file, along with any related functions for parsing them)